tests/foreman/ui/test_gpgkey.py

```diff
@@ -111,16 +123,18 @@ def test_negative_create_2(self, name):
         @assert: gpg key is not created
 
         """
-
-        key_content = read_data_file(VALID_GPG_KEY_FILE)
+        kwargs = {
+            'key_content': self.key_content,
+            'name': name,
+            'org': self.organization.name,
+        }
         with Session(self.browser) as session:
-            make_gpgkey(session, org=self.org_name,
-                        name=name, key_content=key_content)
+            make_gpgkey(session, **kwargs)
             self.assertIsNotNone(self.gpgkey.search(name))
-            make_gpgkey(session, org=self.org_name,
-                        name=name, key_content=key_content)
-            self.assertTrue(self.gpgkey.wait_until_element
-                            (common_locators["alert.error"]))
+            make_gpgkey(session, **kwargs)
+            self.assertTrue(
```

Comment:
Sure, I'll update this assertion. Can you describe which assertions should be changed? I'm still learning my way around the UI test code.